components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/dao/impl/ActionManagementDAOImpl.java

```diff
@@ -106,8 +115,12 @@ public ActionDTO getActionByActionId(String actionType, String actionId, Integer
         if (actionBuilder == null) {
             return null;
         }
-        actionBuilder.setEndpointAndProperties(getActionPropertiesFromDB(actionId, tenantId));
```

Comment:
As we don't use `setEndpointAndProperties()` anymore, let's remove that from the ActionDTOBuilder class

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/6269/commits/dbc0b1d635d2ee72742509dee496d7a8ba755a39